lib/web/auth/ldap/index.js

```diff
@@ -23,15 +23,15 @@ passport.use(new LDAPStrategy({
     tlsOptions: config.ldap.tlsOptions || null
   }
 }, function (user, done) {
-  var uuid = user.uidNumber || user.uid || user.sAMAccountName
-  var username = uuid
+  var uid = user[config.ldap.useridField || 'uid']
```

Comment:
I think we still should fall back to the old defaults because we don't expect our users to change this on update. So old settings shouldn't change anything.